integration/configure_unix_test.go

```diff
@@ -1,5 +1,5 @@
-// +build !windows
-//
+// +build configure configure,!windows
```

Comment:
is this right? or should it be

```suggestion
// +build configure,!windows
```

Reply:
`spaces are OR and commas are AND`

So this reads "include me when specifying configure, or when specifying configure and not windows"

Reply:
TIL 🙌🏽